include/freerdp/peer.h

```diff
@@ -148,7 +147,7 @@ struct rdp_freerdp_peer
 	ALIGN64 psPeerGetEventHandles GetEventHandles;
 	ALIGN64 psPeerAdjustMonitorsLayout AdjustMonitorsLayout;
 	ALIGN64 psPeerClientCapabilities ClientCapabilities;
-	ALIGN64 psPeerComputeNtlmHash ComputeNtlmHash;
```

Comment:
It would be propably better to deprecate that field, and put the new one at then end of the struct (look at FREERDP_DEPRECATED in other places)

Reply:
Sorry no... I'm just not quite sure I understand how that would work. Should we have a second field in `NTLM_CONTEXT` as well? Which would be used when compiling with the DEPRECATED option?

Reply:
IMHO we shall deprecate `ComputeNtlmHash` and have a new field (some "sspi" in the name could make sense) with the right type. And of course have the rest of the code use the new field.

Reply:
OK, got it